specification.md

```diff
@@ -0,0 +1,92 @@
+# Dispute Game Simulation Specifications
+
+This document aims to dial in on a specification and scope for the Primitive team to audit the disputed game with Arbiter simulations. For the purpose of this document we consider a `Claim` to be a move in the dispute game.
+
+## Scope
+We propose to define two core deliverables for the OP labs team. Both deliverables will evolve from this repository.
+- A report of our findings and analysis of the dispute game located in the `security_report` directory.
+
+- A simulation of the dispute game located in the `src` directory. We want to run 100M Sims on a (single or many but takes about 45min to compute) connon trace (on average 1.5b instructions) with at least 5 agent behaviors. The [Simulation Specification](#simulation-specification) section defines the simulation details below.
```

Comment:
100 million sims where each takes 45 minutes? I'm confused. That would take ~8k years to do.

Reply:
For me number of runs is less important than driving the intuition for what levers we have at our disposal in the context of each run.

Reply:
No no no, a single cannon trace (L2 execution trace) takes 45minutes to solve. we can run as many simulations as we like (100M) on a cannon trace.

Reply:
Especially considering we haven't benchmarked how long each run will take, probably wise to just leave any talk of "we will provide results from n runs" out of this